src/collection.js

```diff
@@ -242,6 +242,15 @@ export default class Collection {
   }
 
   /**
+   * Closes any current connection to the database for this collection.
+   *
+   * @return {Promise}
+   */
+  close() {
+    return this.db.close();
+  }
+
+  /**
```

Comment:
We need an 'open' and an 'isOpen' too.

open - returns a promise that resolves when adapter.open does
isOpen - that returns true once the collection is open